mobile-v3/lib/src/app/auth/bloc/auth_bloc.dart

```diff
@@ -4,46 +4,122 @@ import 'package:bloc/bloc.dart';
 import 'package:equatable/equatable.dart';
 import 'package:flutter/cupertino.dart';
 
+import '../../shared/repository/hive_repository.dart';
+
 part 'auth_event.dart';
 part 'auth_state.dart';
 
+// class AuthBloc extends Bloc<AuthEvent, AuthState> {
+//   final AuthRepository authRepository;
```

Comment:
Hi @Karagwa , do we need this commented out code for reference in the near future? If not, could we remove it?